### PR TITLE
Updated os builder to include new export changes

### PR DESCRIPTION
**Pull request by paynejacob**
Changes:
- added configuration for nginx to support uploads
- added crds to chart
- updated osbuilder to latest version

This pr depends on a release including these changes https://github.com/kairos-io/osbuilder/pull/44

### CODE REVIEW

File: charts/osbuilder/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+To access the artifacts generated by your OSArtifact objects from the included nginx server add the following export to
+your OSArtifact at `.spec.exports[]`.
```
Comment (jimmykarily) [inline charts/osbuilder/templates/NOTES.txt]:
```suggestion
your OSArtifact at `.spec.exporter[]`.
```
https://github.com/kairos-io/osbuilder/pull/44/files#diff-762d4e967963055cda9011aa360aa3081320ad92046df2fc0580b7e28174a92aR47

Also, we call it `Exporters` in golang and `exporter` in yaml which makes it a bit confusing.

File: charts/osbuilder/templates/deployment.yaml [hunk 2]
```diff
@@ -48,12 +48,7 @@ spec:
                   - --health-probe-bind-address=:8081
                   - --metrics-bind-address=127.0.0.1:8080
                   - --leader-elect
-                  - --tool-image={{ .Values.image.repository | default "quay.io/kairos/osbuilder-tools" }}:{{ .Values.image.tag | default .Chart.AppVersion }}
-                  - --serve-image={{ .Values.image.serving | default "nginx" }}
-                  - --copy-to-namespace={{ .Release.Namespace }}
-                  - --copy-role=osartifactbuilder-operator-artifactCopier
-                  - --copy-to-pod-label=app.kubernetes.io/name=osbuilder-nginx
-                  - --copy-to-path="/usr/share/nginx/html"
```
Comment (jimmykarily) [inline charts/osbuilder/templates/deployment.yaml:56]: I guess these flags are no longer needed then? Should we remove them from the osbuilder?

File: charts/osbuilder/crds/build.kairos.io_osartifacts.yaml [hunk 3]
```diff
@@ -0,0 +1,8031 @@
+---
```
Comment (jimmykarily) [inline charts/osbuilder/crds/build.kairos.io_osartifacts.yaml]: [We tell people](https://kairos.io/docs/advanced/build/#kubernetes) to install the kairos-crds chart. I think this file should be merged with [this one](https://github.com/kairos-io/helm-charts/blob/main/charts/kairos-crds/templates/osbuilder.yaml)

Comment (paynejacob) [reply charts/osbuilder/crds/build.kairos.io_osartifacts.yaml]: Ill update it to use the existing chart.

File: charts/osbuilder/templates/deployment.yaml (hunk 2 above)
Comment (paynejacob) [reply charts/osbuilder/templates/deployment.yaml:56]: that is a good catch, we can remove them.

File: charts/osbuilder/templates/NOTES.txt (hunk 1 above)
Comment (paynejacob) [reply charts/osbuilder/templates/NOTES.txt]: ill make it consistent

**Comment by jimmykarily** [comment]
Linter failures legit?

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**